lib/html/pipeline/dradis/sanitize.rb

```diff
@@ -0,0 +1,20 @@
+module HTML
+  class Pipeline
+    module Dradis
+      class Sanitize
+        # SEE: https://github.com/gjtorikian/html-pipeline/blob/v2.14.0/lib/html/pipeline/sanitization_filter.rb
+        original_list = HTML::Pipeline::SanitizationFilter::WHITELIST
+
+        ALLOWLIST = original_list.merge(
+          attributes: original_list[:attributes].merge(
```

Comment:
do you need the double #merge ?

I'd expect for the L9 to not be needed, as all the :attributes from the original_list, would already be covered by the L8 merge, no?